sdk/datafactory/arm-datafactory/CHANGELOG.md

```diff
@@ -1,15 +1,225 @@
+  - Interface MongoDbAtlasLinkedService has a new optional parameter driverVersion
+  - Interface ParquetSource has a new optional parameter formatSettings
+  - Interface PipelineExternalComputeScaleProperties has a new optional parameter numberOfExternalNodes
+  - Interface PipelineExternalComputeScaleProperties has a new optional parameter numberOfPipelineNodes
+  - Interface SelfHostedIntegrationRuntime has a new optional parameter selfContainedInteractiveAuthoringEnabled
+  - Interface SelfHostedIntegrationRuntimeStatus has a new optional parameter selfContainedInteractiveAuthoringEnabled
+  - Interface SetVariableActivity has a new optional parameter policy
+  - Interface SetVariableActivity has a new optional parameter setSystemVariable
+  - Interface SqlDWSource has a new optional parameter isolationLevel
+  - Interface SqlMISource has a new optional parameter isolationLevel
+  - Interface SqlServerSource has a new optional parameter isolationLevel
+  - Interface SynapseNotebookActivity has a new optional parameter configurationType
+  - Interface SynapseNotebookActivity has a new optional parameter sparkConfig
+  - Interface SynapseNotebookActivity has a new optional parameter targetSparkConfiguration
+  - Interface WebHookActivity has a new optional parameter policy
+  - Type of parameter type of interface CopySink is changed from "DelimitedTextSink" | "JsonSink" | "OrcSink" | "RestSink" | "AzurePostgreSqlSink" | "AzureMySqlSink" | "AzureDatabricksDeltaLakeSink" | "SapCloudForCustomerSink" | "AzureQueueSink" | "AzureTableSink" | "AvroSink" | "ParquetSink" | "BinarySink" | "BlobSink" | "FileSystemSink" | "DocumentDbCollectionSink" | "CosmosDbSqlApiSink" | "SqlSink" | "SqlServerSink" | "AzureSqlSink" | "SqlMISink" | "SqlDWSink" | "SnowflakeSink" | "OracleSink" | "AzureDataLakeStoreSink" | "AzureBlobFSSink" | "AzureSearchIndexSink" | "OdbcSink" | "InformixSink" | "MicrosoftAccessSink" | "DynamicsSink" | "DynamicsCrmSink" | "CommonDataServiceForAppsSink" | "AzureDataExplorerSink" | "SalesforceSink" | "SalesforceServiceCloudSink" | "MongoDbAtlasSink" | "MongoDbV2Sink" | "CosmosDbMongoDbApiSink" to "DelimitedTextSink" | "JsonSink" | "OrcSink" | "RestSink" | "AzurePostgreSqlSink" | "AzureMySqlSink" | "AzureDatabricksDeltaLakeSink" | "SapCloudForCustomerSink" | "AzureQueueSink" | "AzureTableSink" | "AvroSink" | "ParquetSink" | "BinarySink" | "BlobSink" | "FileSystemSink" | "DocumentDbCollectionSink" | "CosmosDbSqlApiSink" | "SqlSink" | "SqlServerSink" | "AzureSqlSink" | "SqlMISink" | "SqlDWSink" | "SnowflakeSink" | "OracleSink" | "AzureDataLakeStoreSink" | "AzureBlobFSSink" | "AzureSearchIndexSink" | "OdbcSink" | "InformixSink" | "MicrosoftAccessSink" | "DynamicsSink" | "DynamicsCrmSink" | "CommonDataServiceForAppsSink" | "AzureDataExplorerSink" | "SalesforceSink" | "SalesforceServiceCloudSink" | "MongoDbAtlasSink" | "MongoDbV2Sink" | "CosmosDbMongoDbApiSink" | "LakeHouseTableSink"
```

Comment:
This set of lines should be removed as it is the reason that the markdown conversion is not working.